src/seeds.proposals.cpp

```diff
@@ -1082,6 +1082,11 @@ void proposals::addvoice(name user, uint64_t amount) {
   voice_change(user, amount, false, ""_n);
 }
 
+void proposals::questvote (name user, uint64_t amount, bool reduce, name scope) {
+  require_auth(get_self());
+  voice_change(user, amount, reduce, scope);
+}
```

Comment:
How does quest voting affect voice? I think it should probably have its own name space?